src/main/java/controllers/components/CardsMngr.java

```diff
@@ -21,14 +21,18 @@ public CardsMngr() {
     }
 
     public List<Card> getRandSet() {
-        cards = new ArrayList<Card>();
+        ArrayList<Card> playerHand = new ArrayList<Card>(); //create list for 6 cards to be dealt
+        cards = this.getBoardCards(); //get list of cards already created
```

Comment:
This might have a potential flaw since you shallow-copied the list. 
here is a link to an article explaining the difference between shallow copy and deep copy in java.

http://cs-fundamentals.com/java-programming/java-arrays-clone-shallow-deep-copy.php

you might consider using clone() method to get deep copy of the list.

Thank you,
Abdul